lib/voicebase/v2/response.rb

```diff
@@ -2,13 +2,40 @@ module VoiceBase
   module V2
     module Response
 
+      TRANSCRIPT_READY = "finished".freeze
```

Comment:
Minor point, but this variable might be more clear as something like `TRANSCRIPT_READY_STATUS`. As is, I would think it might be a boolean.